### PR TITLE
Write dockerfile to best practices

### PR DESCRIPTION
**Pull request by kjetilk**
Found some stuff in https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#run that I did.

### CODE REVIEW

File: testers/rdf-fixtures/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+    libxml-simple-perl \
+    perl \
+    perlbrew \
+    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -yq `apt-cache depends libattean-perl | awk '/Depends:/{print$2}'`
+RUN apt-get update && apt-get install -yq `apt-cache depends liburi-namespacemap-perl | awk '/Recommends:/{print$2}'`
 RUN perlbrew install-cpanm
 RUN /root/perl5/perlbrew/bin/cpanm install Test::FITesque::RDF
-RUN apt-get install -yq git
+RUN apt-get update && apt-get install -yq git
```
Comment (michielbdejong) [inline testers/rdf-fixtures/Dockerfile]: you can move this one into the big list above as well

File: testers/rdf-fixtures/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+    libplack-perl \	   
+    libtest-lwp-useragent-perl \	
+    libtest-modern-perl \
+    libtest-rdf-perl \
+    libtypes-uri-perl \
+    libxml-parser-perl \
+    libxml-simple-perl \
+    perl \
+    perlbrew \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (michielbdejong) [inline testers/rdf-fixtures/Dockerfile]: I've also seen `apt-get clean` here, but no strong opinion

Comment (kjetilk) [reply testers/rdf-fixtures/Dockerfile]: It actually mentions that near the end, it is done automatically, but you need the rm to remove it from the package.

File: testers/rdf-fixtures/Dockerfile (hunk 1 above)
Comment (kjetilk) [reply testers/rdf-fixtures/Dockerfile]: Yeah, but I figured it could stay since we might take it from CPAN at some point.

**Pull request merged**